### PR TITLE
Update app_onc.py

### PR DESCRIPTION
**Pull request by dterach**
Commented out SEDkit imports

### CODE REVIEW

**Comment by Will-Cooper** [comment]
We are getting rid of sedkit because it requires healpy which breaks on windows?
Note removing the imports will still have it break later when it tries to call those

**Comment by dterach** [comment]
> We are getting rid of sedkit because it requires healpy which breaks on windows?
> Note removing the imports will still have it break later when it tries to call those

Yes, from what I read healpy isn't compatible on Windows. I just started with the inputs so that there was an initial commit. I ran into an issue or 2 after attempting to comment out some more, so I didn't want to make any new commits yet.

File: onc_app/app_onc.py [hunk 1]
```diff
@@ -926,4 +927,4 @@ def onc_feedback():
```
Comment (Will-Cooper) [inline onc_app/app_onc.py:927]: We can probably delete this file once we're happy with the ```app_simple.py```

File: simple_app/app_simple.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+from astrodbkit2.astrodb import Database
+#connection_string = 'sqlite:///bdnyc_database.db'   #Had been 'sqlite:///SIMPLE.db'
+#db_dir = r'C:\Users\danie\SIMPLE-web'
+#db = Database(connection_string)
+#db.load_database(db_dir)
+
+db_file = 'sqlite:///SIMPLE.db'
+db = Database(db_file)
+pd.set_option('max_colwidth', -1)
```
Comment (Will-Cooper) [inline simple_app/app_simple.py:32]: noticed this was leaving a warning, what is it doing?

File: simple_app/app_simple.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+#@app_onc.route('/')
+#@app_onc.route('/index')
+
+@app_simple.route('/')
+@app_simple.route('/index')
+
+# Page with a text box to take the SQL query
+#@app_onc.route('/index', methods=['GET', 'POST'])
+
+@app_simple.route('/index', methods=['GET', 'POST'])
```
Comment (Will-Cooper) [inline simple_app/app_simple.py:48]: is this overwriting the previous ```@app_simple.route```, if so why do we have those lines?

File: simple_app/templates/index_simple.HTML [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<div class='block'>
+
+        <h1>Welcome!</h1>
+        <p style="line-height:1.5em;">The Orion Nebula Cluster Database is a cross-matched and curated collection of<br>
```
Comment (Will-Cooper) [inline simple_app/templates/index_simple.HTML:8]: Can update text for SIMPLE

**Review by Will-Cooper — COMMENTED** [review]
I won't request the changes but I can see a lot of redudant bits here, e.g. all the static images used in ONC. Also script files like app_onc.py. Will leave it up for debate if we want to remove files like that now, or raise an issue for later handling. cc. @dr-rodriguez @kelle.

**Review by Will-Cooper — APPROVED** [review]

**Pull request merged**